File: go/wam/3522.go
const WamPrivacyHighlightDaily = 3522 // channel: regular

type PrivacyHighlightDailyEvent struct {
	DialogAppearCount int64 `wam:"dialog_appear_count"` // field 1
	DialogSelectCount int64 `wam:"dialog_select_count"` // field 2
	NarrativeAppearCount int64 `wam:"narrative_appear_count"` // field 3
	PrivacyHighlightCategory PRIVACYHIGHLIGHTCATEGORYENUM `wam:"privacy_highlight_category"` // field 4
	PrivacyHighlightSurface PRIVACYHIGHLIGHTSURFACEENUM `wam:"privacy_highlight_surface"` // field 5
}

File: ts/wam/3522.ts
export const WamPrivacyHighlightDaily = 3522 as const

export interface PrivacyHighlightDailyEvent {
  /** field 1, wire `dialog_appear_count` */
  dialogAppearCount?: number
  /** field 2, wire `dialog_select_count` */
  dialogSelectCount?: number
  /** field 3, wire `narrative_appear_count` */
  narrativeAppearCount?: number
  /** field 4, wire `privacy_highlight_category` */
  privacyHighlightCategory?: PRIVACYHIGHLIGHTCATEGORYENUM
  /** field 5, wire `privacy_highlight_surface` */
  privacyHighlightSurface?: PRIVACYHIGHLIGHTSURFACEENUM
}